plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/ServicePortDialog.java

```diff
@@ -140,26 +141,23 @@ public IStatus validate(Object value) {
 	}
 	class PodPortValidator implements IValidator{
 		
-		private final int MAXLENGTH = 63;
-		private final Pattern REGEXP = Pattern.compile("[a-z0-9]([a-z0-9-]*[a-z0-9])*");
-
-		private final IStatus ERROR = ValidationStatus.error("The target port must be at most 15 characters, matching regex [a-z0-9]([a-z0-9-]*[a-z0-9])*, and hyphens cannot be adjacent to other hyphens): e.g. \"http\"");
```

Comment:
as far as I understood ports can be numerical but also alphanumeric names? https://issues.jboss.org/browse/JBIDE-21761?focusedCommentId=13169586&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13169586